pants-plugins/internal_plugins/releases/register.py

```diff
@@ -181,7 +181,11 @@ async def check_default_tools(
                     SessionValues(
                         {
                             OptionsBootstrapper: OptionsBootstrapper(
-                                tuple(), ("./pants",), args, Config(tuple()), CliAlias()
+                                tuple(),
+                                ("./pants",),
```

Comment:
Just noting in general, perhaps for another issue/pr: This should maybe be `"pants"` instead, being the recommended way to run Pants now.

Reply:
Yeah, but I should revert this change either way. :-)